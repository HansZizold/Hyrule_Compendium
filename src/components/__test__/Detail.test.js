import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import ItemDetailed from '../Detail';
import store from '../../redux/configureStore';

describe(' Items Component Rendering ', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <ItemDetailed />
        </Provider>,
      );
    expect(tree).toMatchSnapshot();
  });
});
