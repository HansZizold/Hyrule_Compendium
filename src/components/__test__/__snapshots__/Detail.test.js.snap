// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Items Component Rendering  renders correctly 1`] = `null`;
